rcgen/src/ring_like.rs

```diff
@@ -35,5 +39,5 @@ pub(crate) fn rsa_key_pair_public_modulus_len(kp: &signature::RsaKeyPair) -> usi
 	}
 }
 
-#[cfg(not(any(feature = "ring", feature = "aws_lc_rs")))]
+#[cfg(all(feature = "crypto", not(any(feature = "ring", feature = "aws_lc_rs"))))]
 compile_error!("At least one of the 'ring' or 'aws_lc_rs' features must be activated");
```

Comment:
maybe the error could be improved to suggest the alternative of disabling the crypto feature?